daemon/src/model/cfd.rs

```diff
@@ -610,6 +608,10 @@ impl Cfd {
         self.settlement_proposal.is_some()
     }
 
+    fn is_in_non_collaborative_settlement(&self) -> bool {
```

Comment:
`is_in_force_close()`? not sure whether we've reached full agreement on the naming yet, but last I checked we were leaning towards "close" and "force-close"